sql/hive/src/test/scala/org/apache/spark/sql/hive/test/TestHive.scala

```diff
@@ -685,3 +685,10 @@ private[hive] object HiveTestJars {
     targetFile
   }
 }
+
+private[hive] object TestHiveUtils {
+  def withHiveExternalCatalog(sc: SparkContext): SparkContext = {
```

Comment:
Could you move this method into `TestHiveContext`?

Reply:
> Could you move this method into `TestHiveContext`?

Sure, more suitable place.